src/k8s-extension/azext_k8s_extension/_help.py

```diff
@@ -83,41 +83,79 @@
     short-summary: Commands to discover Kubernetes Extension Types.
 """
 
-helps[f'{consts.EXTENSION_NAME} extension-types list'] = f"""
+helps[f'{consts.EXTENSION_NAME} extension-types get-by-cluster'] = f"""
     type: command
-    short-summary: List Kubernetes Extension Types.
+    short-summary: Get properties for a Cluster Extension Type for an existing cluster.
     examples:
-      - name: List Kubernetes Extension Types
+      - name: Get properties for a Cluster Extension Type for an existing cluster
         text: |-
-          az {consts.EXTENSION_NAME} extension-types list --resource-group my-resource-group \
+          az {consts.EXTENSION_NAME} extension-types get-by-cluster --resource-group my-resource-group \
+--cluster-name mycluster --cluster-type connectedClusters --extension-type microsoft.openservicemesh
+"""
+
+helps[f'{consts.EXTENSION_NAME} extension-types get-by-location'] = f"""
+    type: command
+    short-summary: Get properties for a Cluster Extension Type in a region.
+    examples:
+      - name: Get properties for a Cluster Extension Type in a region
+        text: |-
+          az {consts.EXTENSION_NAME} extension-types get-by-location --location eastus --extension-type microsoft.openservicemesh
+"""
+
+helps[f'{consts.EXTENSION_NAME} extension-types get-version-by-cluster'] = f"""
+    type: command
+    short-summary: Get properties associated with a Cluster Extension Type version for an existing cluster.
```

Comment:
>xisting cluster [](http://example.com/codeflow?start=91&length=15)

Here too, clarify which properties of the cluster are used in filtering the versions